bootstrap.js

```diff
@@ -192,7 +192,9 @@
         var script = document.createElement("script");
         script.src = location;
         script.onload = function () {
-            if(loadCallback) loadCallback(script);
+            if(loadCallback) {
```

Comment:
Not jshint did not complain. I will do apply this keyword space rule in the future.